README.md

```diff
@@ -34,6 +49,70 @@ The following verification utility will re-verify these numbers.
 just verify
 ```
 
+### Setting up the environment
+
+Before setting up your environment, please ensure that you have all the prerequisite steps above fulfilled.
+
+To get started, populate the `.env` file with your information, such as your Google Voice number and Twitter tokens. For example:
+
+```bash
+BOT_NUMBER=<Google Voice number>
+ADMIN_CONTACT=<Your number>
+TWITTER_API_KEY=<Generated Twitter API Key>
+TWITTER_API_SECRET=<Generated Twitter API Secret>
+TWITTER_ACCESS_TOKEN=<Generated Twitter Access Token>
+TWITTER_TOKEN_SECRET=<Generated Twitter Access Token Secret>
+...
+
+```
+
+Note that to you do not need to fill out all of the environment variables yet. Many of these variables will have defaults. To use the defaults, simply remove the variable line. For example, if you would like to use the default timezone, you would remove the entire line:
+
+`DEFAULT_TZ=TZ database formatted name for a timezone. Defaults to US/Pacific`
+
+Once you have your phone numbers in the `.env` file, run `just register`.
+
+**WINDOWS USERS:** If you are on Windows, you can pass this into your `justfile` to allow it to use Powershell instead of using `sh`:
+
+```
+# use PowerShell instead of sh:
+set shell := ["powershell.exe", "-c"]
```

Comment:
I always forget about Windows, sorry! :sweat_smile: We should probably just use the [`os_family` function provided in `just`](https://github.com/casey/just#system-information) to make this easier. We could add the following line to the top of our `justfile` so users don't have to make this change themselves:
```
set shell := if os_family() == "unix" { ["sh", "-cu"] } else { ["powershell", "-cu"] }
```

Reply:
Son of a bitch. I spent like an hour trying to set that and couldn't make it happen for some reason lol. Glad it's this easy.

Reply:
Ha oof, I love `just` but it could definitely be documented better

Reply:
Oh that's cool! Happy to add that and open a new PR for this change.